### ISSUE TITLE
Using information from insertions/deletions in Cluster (phase) alignments 

### ISSUE THREAD

**Issue by ollenordesjo** [igvteam/igv#965]
Just making a separate issues for something I mentioned here: https://github.com/igvteam/igv/issues/960.

I think the clustering/phasing feature could become very effective if also including insertions/deletions to be informative of group membership somehow.

Thanks!

**Comment by jrobinso** [comment]
@ollenordesjo Judging a "match" between indels in 3rd gen data is difficult because there is so much noise.  Often I see a pileup of indels that look like they might be related but no 2 of them are the same length, for example.   Clearly insertions should also match sequence, but how closely, if it's any length at ll there will be mismatches?   I ran into these types of issues when trying to include indels in consensus filtering, but did not resolve them.

**Comment by ollenordesjo** [comment]
Absolutely, it's definitely not trivial. Do you think it would be painful/brittle to do something similar to aliview, which links up mafft/muscle to do MSAs? I guess that would at least it's down to the user to parametrise mafft/muscle instead of having to assume to much about the data in IGV?

**Comment by jrobinso** [comment]
@ollenordesjo I don't know, as I don't know what aliview, mafft/muscle, or MSAs are.

**Comment by ollenordesjo** [comment]
Happy to give some pointers, definitely don't feel pressured to make this a feature, but just for the purpose of explaining it clearer:

Here's an example of how aliview is allowing the user to use external programs to do multiple sequence alignments, that's what I was suggesting in terms of linking up to mafft/muscle.
![image](https://user-images.githubusercontent.com/75309879/119502328-bc167d80-bd61-11eb-9cfc-0e3b8ebab115.png)

After running mafft or muscle, the sequences will typically be organised similar to this in the UI:
![image](https://user-images.githubusercontent.com/75309879/119502589-fe3fbf00-bd61-11eb-987f-bfe684bcd832.png)

Hope that clarifies things!
